File: Tune/app/src/main/java/com/example/tune/activities/models/Comment.java
package com.example.tune.activities.models;

import java.io.Serializable;

public class Comment implements Serializable {

    private User user;
    private String content;

    public Comment(User user, String content) {
        this.user = user;
        this.content = content;
    }

    public User getUser() {
        return user;
    }

    public String getContent() {
        return content;
    }
}

File: tune_kotlin/app/src/main/java/com/example/tune_kotlin/models/Comment.kt
package com.example.tune_kotlin.models

import com.google.firebase.auth.FirebaseUser
import java.io.Serializable

class Comment() : Serializable {
    var id: Long? = null
    var user: String? = null
    var content: String? = null
    var date: String? = null

    constructor(user: String?, content: String?, date: String?): this(){
        this.user = user
        this.content = content
        this.date = date
    }
}